plugins/colorschemes/catppuccin.nix

```diff
@@ -144,21 +144,52 @@ in {
       integrations = {
         aerial = helpers.defaultNullOpts.mkBool false "";
 
+        alpha = helpers.defaultNullOpts.mkBool true "";
+
         barbar = helpers.defaultNullOpts.mkBool config.plugins.barbar.enable "";
 
+        barbecue = {
+          dim_dirname = helpers.defaultNullOpts.mkBool true "";
+
+          bold_basename = helpers.defaultNullOpts.mkBool true "";
+
+          dim_context = helpers.defaultNullOpts.mkBool false "";
+
+          alt_background = helpers.defaultNullOpts.mkBool false "";
+        };
+
         beacon = helpers.defaultNullOpts.mkBool false "";
 
+        # TODO: bufferline = ;
```

Comment:
You still have a few TODOs left

Reply:
These TODOs are special integrations which require additional setup in other plugins. I need help to implement them

Reply:
Ok, let's leave them as is for now.